lib/platform/gitea/index.ts

```diff
@@ -84,6 +85,11 @@ function toRenovatePR(data: helper.PR): Pr | null {
     return null;
   }
 
+  const createdBy = data.user?.username;
+  if (createdBy && botUserName && createdBy !== botUserName) {
```

Comment:
So if either botUserName or createdBy are undefined then we'll pass the PR up?

Reply:
Yes, I aimed not to break existing logic if something went wrong with those fields. Maybe it would be good to also log some message.